app/models/weam.rb

```diff
@@ -165,12 +165,11 @@ def approved?
     return false unless poo_status =~ Regexp.new('aprvd', 'i')
 
     # VET TEC ONLY
-    return vet_tec_flags_for_approved? if applicable_law_code =~ Regexp.new("#{ALCVT}", 'i')
-    
+    return vet_tec_flags_for_approved? if applicable_law_code =~ Regexp.new(ALCVT.to_s, 'i')
```

Comment:
I learned from [another PR](https://github.com/department-of-veterans-affairs/gibct-data-service/pull/392#discussion_r306017677) that this is a linting correction.  `ALCVT` is already a string, so you don't need to call `to_s`.  The regex can just be `Regexp.new(ALCVT, 'i')`